workflows/offline_ml_diags/offline_ml_diags/compute_diags.py

```diff
@@ -422,7 +424,17 @@ def main(args):
             os.path.join(args.output_path, "model_sensitivity_figures"),  # type: ignore
         )
     except AttributeError:
-        pass
+        try:
+            input_feature_indices = get_variable_indices(
+                data=pred_mapper[timesteps[0]], variables=model.input_variables
+            )
+            plot_rf_feature_importance(
```

Comment:
Ideally we shouldn't be creating plots during the `compute_diags.py` script. Can we just compute the data necessary to make this plot and save it to be loaded/plotted in the report script? This would be really helpful if, for example, we want to load and compare the diagnostic for a couple different models.

Reply:
For these cases since they require the model (which the report step doesn't load) and the calculation of the input sensitivities is a one liner for the keras models (`model.jacobian`) and will be a one liner for the RF as well after Jeremy's fv3fit refactoring. In the case where we want compare these for multiple models in a notebook it is currently not too bad to just load the models and directly calculate the input sensitivities in the notebook rather than save the sensitivity diagnostic in an intermediate format that has to be opened and parsed by the report script. I do agree that ideally these should be treated like the other diagnostics. That would require a fair amount of changes here as well as the report, so to keep this PR scope limited I will leave the plot saving as it is here, and open an issue to resolve in a follow on PR. Let me know if that sounds ok @oliverwm1 @frodre

Reply:
Okay sounds good. I don't really know what the code is like to compute the sensitivities, so thought it might be nice to be able to directly open the results in a notebook. Definitely agree no changes required for this PR! But if you agree it's worth the effort, would be nice to address in the future.